test/common/ssl/context_impl_test.cc

```diff
@@ -155,16 +210,66 @@ TEST_F(SslContextImplTest, TestGetCertInformation) {
   // For the cert_chain, it is dynamically created when we run_envoy_test.sh which changes the
   // serial number with
   // every build. For cert_chain output, we check only for the certificate path.
-  std::string ca_cert_partial_output(TestEnvironment::substitute(
-      "Certificate Path: {{ test_rundir }}/test/common/ssl/test_data/ca_cert.pem, Serial Number: "
-      "eaf3b0ea1d0e579a, "
-      "Days until Expiration: "));
-  std::string cert_chain_partial_output(
-      TestEnvironment::substitute("Certificate Path: {{ test_tmpdir }}/unittestcert.pem"));
+  std::string ca_cert_json = R"EOF({
+ "path": "{{ test_rundir }}/test/common/ssl/test_data/ca_cert.pem",
+ "serial_number": "eaf3b0ea1d0e579a",
+ "subject_alt_names": [],
+)EOF";
+
+  std::string cert_chain_json = R"EOF({
+ "path": "{{ test_tmpdir }}/unittestcert.pem",
+)EOF";
+
+  std::string ca_cert_partial_output(TestEnvironment::substitute(ca_cert_json));
+  std::string cert_chain_partial_output(TestEnvironment::substitute(cert_chain_json));
+  EXPECT_TRUE(MessageUtil::getJsonStringFromMessage(*context->getCaCertInformation(), true, true)
+                  .find(ca_cert_partial_output) != std::string::npos);
+  EXPECT_TRUE(MessageUtil::getJsonStringFromMessage(*context->getCertChainInformation(), true, true)
+                  .find(cert_chain_partial_output) != std::string::npos);
+}
+
+TEST_F(SslContextImplTest, TestGetCertInformationWithSAN) {
+  std::string json = R"EOF(
```

Comment:
This is v1 config format to be removed soon, can you use v2 format?

Reply:
I can use for these two tests. But rest of the tests are still using v1 config in this same file. I prefer to leave these two also in v1 for now and cleanup all of them to v2 in a separate PR if that is Ok.